go.mod

```diff
@@ -0,0 +1,208 @@
+module istio.io/istio
```

Comment:
To confirm: is this file auto-generated ? And how is it updated ?
Did anyone compare the versions with what go dep was producing - are we picking similar stable versions ?

Just had to fight with Kind build breakages due to moving to go modules - it would be nice to give people more time to review this kind of massive change, and include some info about the testing.

Since we still use vendor - I hope build speed will not regress.

Reply:
Based on kuat's earlier message, it appears it was generated with `go mod init`:

> go mod init automatically translates the required information from dep, glide, govendor, godep and 5 other pre-existing dependency managers into a go.mod file that produces the equivalent build.

I would have hoped that 'equivalent builds' means that we are using the same versions. But this apparently also pulled in a newer gRPC version.

Reply:
The reason why gRPC was updated is because dep did not respect module requirements, and some modules requested a newer version than dep chose.